src/runtime/markdown-parser/handler/utils.ts

```diff
@@ -20,11 +20,13 @@ export function parseThematicBlock (lang: string) {
   const language = lang.replace(/[{|[](.+)/, '').match(/^[^ \t]+(?=[ \t]|$)/)
   const highlightTokens = lang.match(/{([^}]+)}/)
```

Comment:
```suggestion
  const highlightTokens = lang.match(/{([\s\d\-,]+)}/)
```
It might be a good idea to prevent this from matching things that can't be parsed by `parseHighlightedLines` (new L33)
(Maybe also make content optional with `*` instead of `+` to be able to not provide highlights, but still use `{…}` in `meta`)